regolith/helpers/attestationshelper.py

```diff
@@ -123,21 +122,19 @@ def db_updater(self):
         for person in people:
             person_appts = person.get('appointments', None)
             if person_appts:
-                for item in person_appts:
-                    for appt_value_list in item.values():
-                        for p_appt in appt_value_list:
-                            grantid = p_appt.get('grant')
-                            if grantid == grant_id:
-                                #accountnr = grant.get('account', grant['alias'])
-                                loading = p_appt.get('loading')
-                                bd = get_dates(p_appt).get("begin_date")
-                                begin = min(begin, bd)
-                                ed = get_dates(p_appt).get("end_date")
-                                end = max(end, ed)
-                                months_on_grant = (ed - bd).days / 30.4 * loading
-                                appt = (person['_id'], bd, ed, loading, months_on_grant)
-                                appts.append(appt)
-            #                    on_grant = []
+                for _id, p_appt in person_appts.items():
+                    grantid = p_appt.get('grant')
+                    if grantid == grant_id:
+                        #accountnr = grant.get('account', grant['alias'])
```

Comment:
delete things we don't use.  You can do this at the end when we are sure we don't need it.